app/Rules/EntityFileRule.php

```diff
@@ -48,7 +48,7 @@ public function passes($attribute, $value)
             return false;
         }
 
-        return !(!in_array($value->getClientOriginalExtension(), ['mp3', 'ogg', 'json']))
+        return !(!in_array($value->getClientOriginalExtension(), ['mp3', 'ogg', 'json', 'csv']))
```

Comment:
Does csv detection not work on the guessExtension() line? Because getClientOriginalExtension is less secure (basically looks at the file name)